campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -266,6 +258,15 @@
                 var btnSelected;
                 var amount;
                 var presets = '{{donationAmountFirst}},{{donationAmountSecond}},{{donationAmountThird}},{{donationAmountFourth}}';
+                var numberFormat = new Intl.NumberFormat('{{locale}}', {
```

Comment:
also btw `{{ locale }}` with spaces between is more readable and consistent with the rest variables.